gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -286,6 +286,17 @@ public GsonBuilder disableInnerClassSerialization() {
     return this;
   }
 
+  /**
+   * Configures Gson to exclude anonymous and local classes during serialization.
+   *
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   * @since 2.8.2
+   */
+  public GsonBuilder disableAnonymousAndLocalClassSerialization() {
```

Comment:
Afaik this is already disallowed by design.
If you're trying to add it as a feature it should have its own pull request, since it's a completely separate change.

Reply:
@SimonVT , thanks for your review. I agree with you that we can break this into two PRs.

I agree that this variable being inverse. @anirudhramanan let us fix this.

Reply:
@SimonVT thanks for pointing it out. It has been fixed